### DISCUSSION TITLE
Does Quasar have something to animate a number counter?

### DISCUSSION THREAD

**Discussion by BenJackGill** [quasarframework/quasar#14792] [General - Components / Directives / etc]
I have a number variable and in my template I want to animate it counting up from 0.

Does Quasar have a utility or component or something for that?

I can always use another Vue package or even plain JavaScript but thought I'd check here first in case there is a Quasar way of doing it.

Have also looked through the docs and didn't find anything but I could be missing something.

**Comment by ecotrissy** [comment]
You might find [Animate.css](https://animate.style/) helpful.
See https://quasar.dev/options/animations
